solutions/cf-609E.mdx

```diff
@@ -0,0 +1,182 @@
+---
+id: cf-609E
+source: CF
+title: Minimum spanning tree for each edge
+author: Chongtian Ma
+---
+
+[Official Analysis](https://codeforces.com/blog/entry/22187)
+
+## Implementation 
+
+As mentioned in the official editorial, the answer for each edge is removing the heaviest edge in the MST in
+the path from vertex $u$ to $v$ and add the given weight. This will ensure that the MST is still connected.
```

Comment:
if it's already mentioned in the editorial, why mention it here?

at least add some extra intuition/insight

also move out of implementation

Reply:
is it better now?